server/src/main/java/io/spine/server/EnvSetting.java

```diff
@@ -194,10 +227,28 @@ void reset() {
      * @param type
      *         the type of the environment
      */
-    void use(V value, Class<? extends EnvironmentType> type) {
+    public void use(V value, Class<? extends EnvironmentType> type) {
```

Comment:
Maybe we could return `this`? For call chaining, that is.

I am asking since the example from the PR description is a bit mouthful. I think I would look better like this:
```java

    EnvSetting<UserReader> setting = new EnvSetting<>();
    setting.use(projectsUserReader(), Production.class)
           .use(localUserReader(), Local.class)
           .use(testsUserReader(), Tests.class);
```